src/test/java/org/folio/circulation/domain/ConfigurationServiceTest.java

```diff
@@ -0,0 +1,67 @@
+package org.folio.circulation.domain;
+
+import api.support.builders.ConfigurationBuilder;
+import api.support.builders.TimeZoneConfigBuilder;
+import io.vertx.core.json.JsonObject;
+import org.joda.time.DateTimeZone;
+import org.junit.BeforeClass;
+import org.junit.Test;
+
+import java.util.Collections;
+
+import static org.junit.Assert.assertEquals;
+
+public class ConfigurationServiceTest {
+
+  private static ConfigurationService service;
+
+  @BeforeClass
+  public static void before() {
+    service = new ConfigurationService();
+  }
+
+  @Test
+  public void testUtcTimeZone() {
+    String timeZoneValue = "{\"locale\":\"en-US\",\"timezone\":\"UTC\"}";
```

Comment:
I'd preferred that JSON wasn't created in hard coded strings, as I find this is brittle to changes, and repetitive.

Personally, I'd use something similar to the mechanism I proposed in the ConfigurationExample class.

Reply:
Fixed